tests/a_scripts/inference/test_convert_composer_to_hf.py

```diff
@@ -1563,6 +1563,8 @@ def test_mptmoe_huggingface_conversion_callback(
 
             # Check output equivalence
             loaded_model = loaded_model.cuda().bfloat16()  # type: ignore
+            for k, v in batch.items():
+                batch[k] = v.cuda()
```

Comment:
i'm curious, did you figure out the root cause of this?

Reply:
not really, didn't investigate further. did confirm the batch was not on device at this point.